projects/graph/src/graph.py

```diff
@@ -5,5 +5,6 @@
 
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
+
     def __init__(self):
-        pass  # TODO
+        self.graph = {}
```

Comment:
Interesting, why the choice to name this attribute self.graph?

Reply:
I didn't read instructions clearly yet and just wanted to push up an initial commit.

It has been changed since. 😄